kolibri/plugins/learn/assets/src/vue/content-page/index.vue

```diff
@@ -37,23 +41,44 @@
   const constants = require('../../state/constants');
   const getters = require('../../state/getters');
   const ContentKinds = require('kolibri.coreVue.vuex.constants').ContentKinds;
+  const UserKinds = require('kolibri.coreVue.vuex.constants').UserKinds;
 
   module.exports = {
     $trNameSpace: 'learnContent',
     $trs: {
       recommended: 'Recommended',
+      nextContent: 'Next Content',
     },
     computed: {
-      canDownload() { return this.kind !== ContentKinds.EXERCISE; },
+      canDownload() { return this.content.kind !== ContentKinds.EXERCISE; },
+      showNextBtn() { return this.content.kind === ContentKinds.EXERCISE; },
       recommendedText() {
         return this.$tr('recommended');
       },
+      progress() {
+        return this.userkind.includes(
+          UserKinds.LEARNER) ? this.summaryProgress : this.sessionProgress;
+      },
```

Comment:
the whitespace in this function is a little messy because the `(` and `)` don't line up.

Maybe:

```javascript
if (this.userkind.includes(UserKinds.LEARNER)) {
  return this.summaryProgress;
}
return this.sessionProgress;
```